src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsConnectionMiddlewareTests.cs

```diff
@@ -796,6 +796,7 @@ void ConfigureListenOptions(ListenOptions listenOptions)
         [ConditionalFact]
         [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10)] // HTTP/2 requires Win10
         [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "ALPN not supported")]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/34562", "Windows.81.Amd64.Open")]
```

Comment:
Revert these already resolved in main https://github.com/dotnet/aspnetcore/commit/4c0cb774815a334b4ff389911477ab7ae821b163

Reply:
done